webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTableSchema.js

```diff
@@ -0,0 +1,100 @@
+import {
+  sortableHeadFormat,
+  nameFormat,
+  actionsFormat,
+  hostsCountFormat,
+} from './ModelsTableFormats';
+
+export const schema = ({
+  getModels, url, sortBy, sortOrder,
+}) => [
+  {
+    header: {
+      label: __('Name'),
+      props: {
+        className: 'col-md-4',
+        index: 0,
+      },
+      formatters: [sortableHeadFormat(getModels, url, sortBy, sortOrder)],
+    },
+    cell: {
+      props: {
+        className: 'ellipsis',
```

Comment:
afair we have an ellipsis component? @amirfefer ?

Reply:
That's true, but looking at the DOM, it is different: now `<td>` has `ellipsis` in its class while using the  component a `<div>` is wrapped inside the `<td>` which wraps the value. If that is not a problem, I can use the component with no problems. :+1: